rfcs/0051-mark-stale-issues.md

```diff
@@ -0,0 +1,82 @@
+---
+feature: mark-stale-issues
+start-date: 2019-08-24
+author: Ryan Mulligan
+co-authors: (find a buddy later to help our with the RFC)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Mark stale Nixpkgs issues and pull requests (hereafter both referred
+to as simply "issues") on GitHub using an application provided by
+GitHub.
+
+# Motivation
+[motivation]: #motivation
+
+The Nixpkgs GitHub page has a large number of open issues causing
+community angst and misrepresenting the responsiveness of the project.
```

Comment:
Nitpicking a bit, I would prefer if the motivation would be to reduce the number of open issues so that we can focus on the important ones. Something like:

> We have a large number of open issues and PRs that have accumulated over the years. Not all of them are still valid and needs our attention.

Reply:
Thanks; that sounds a lot more positive.